src/datasets/features/audio.py

```diff
@@ -84,6 +87,10 @@ def decode_example(self, value: dict) -> dict:
         Returns:
             dict
         """
+        if not self.decode:
+            raise RuntimeError(
+                "Decoding is disabled for this feature. Set the `decode` attribute to `True` to enable it."
```

Comment:
(nit) "Set the `decode` attribute to `True` to enable it" can make the user think that they can just do
```python
datasets.features[name].decode = True
```
which is not allowed since it doesn't update the `fingerprint` nor `dataset.features._column_requires_decoding`

Maybe we can rephrase this to something like this ?
```suggestion
                "Decoding is disabled for this feature. Please use Audio(decode=True) instead."
```

the same applied for Image as well